src/components/switches/radio/README.md

```diff
@@ -40,11 +40,12 @@ import { Radio } from '@commercetools-frontend/ui-kit/switches';
 
 ##### Radio.Option
 
-| Props        | Type     | Required | Values | Default | Description                                                   |
-| ------------ | -------- | :------: | ------ | ------- | ------------------------------------------------------------- |
-| `children`   | `string` |    -     | -      | -       | The descriptive text of the Radio.Option                      |
-| `isDisabled` | `bool`   |    -     | -      | `false` | Disables the Radio.Option within the Radio.Group              |
-| `value`      | `string` |    ✅    | -      | -       | The value of the option passed to `onChange` in `Radio.Group` |
+| Props        | Type     | Required | Values | Default | Description                                                                                                      |
+| ------------ | -------- | :------: | ------ | ------- | ---------------------------------------------------------------------------------------------------------------- |
+| `children`   | `string` |    -     | -      | -       | The descriptive text of the Radio.Option                                                                         |
+| `isDisabled` | `bool`   |    -     | -      | `false` | Disables the Radio.Option within the Radio.Group                                                                 |
+| `isHovered`  | `bool`   |    -     | -      | `false` | Renders the Radio.Option in a hovered state. We need this, as hover can somethimes come from a parent component. |
```

Comment:
As far as I understand, the `isHovered` passed in here adds to the natural hovered state which uses `:hover`. So if any of them are true, the hovered state is shown, right?

The current documentation makes it seem as-if we always need to pass `isHovered`, when it is actually only needed seldom. I would change the wording to make it more obvious that this is an edge-case to prevent devs from reimplementing the `isHovered` for the case where the component supports it out of the box.

I would also add a note about this in the propTypes of the component's source code itself (not the docs).